access.tex

```diff
@@ -17,7 +17,7 @@ \subsection{Rubin Science Platform}
 It comprises three different ``Aspects'': a \emph{Portal} Aspect designed to provide an environment for data discovery, query, filtering, and visualization, a \emph{Notebook} Aspect to enable next-to-the-data analysis, and an \emph{API} Aspect that enables programmatic access to the Rubin data products via Virtual Observatory (VO) interfaces.
 The Portal and Notebook Aspects of the RSP make use of the same APIs as the API Aspect to internally access the LSST datasets.
 
-An early version of the RSP is currently running at the US Data Access Center (US DAC) that provides access to provides DP0.1 and DP0.2. 
+An early version of the RSP is currently running at the US Data Access Center (US DAC) that provides access to Data Preview 0. 
```

Comment:
I've corrected a typo here and removed the specificity of reference to 0.1 and 0.2.  I think it's OK to say "running at the US Data Access Center" even though it blurs some details of differences between the DP0 data service architecture and the ultimate hybrid architecture.